app-android/src/main/java/com/ustadmobile/port/android/view/ScopedGrantDetailFragment.kt

```diff
@@ -109,4 +122,48 @@ class ScopedGrantDetailFragment: UstadDetailFragment<ScopedGrantWithName>(
         }
 
     }
+}
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun ScopedGrantDetailScreen(
+    uiState: ScopedGrantDetailUiState,
+    onItemClick: (BitmaskFlag) -> Unit = {}
+){
+
+    Column (
+        modifier = Modifier
+            .fillMaxSize()
+    ){
+        uiState.bitmaskList.forEach { bitmask ->
+            ListItem(
+                modifier = Modifier.clickable {
+                    onItemClick(bitmask)
+                },
+                text = {
+                    Text(messageIdResource(id = bitmask.messageId))
+                }
+            )
```

Comment:
Please add a trailing icon to show a check or a cross (e.g. check if it is enabled, cross if not enabled).